applications/GeoMechanicsApplication/tests/Interface/README.md

```diff
@@ -0,0 +1,86 @@
+-   Density of $7850.0 \  \mathrm{[kg/m^3]}$.
+-   Cross area of $0.01 \  \mathrm{[m^2]}$,
+-   Moment of inertia about Z axis $I33 = 8.33333\cdot 10^{-8} \  \mathrm{[kg m^2]}$
+
+The interface is described with SmallStrainUDSM2DInterfaceLaw and it has a cohesion of $1000 \  \mathrm{[kN]}$ and a stiffness of $1e12 \  \mathrm{[Pa]}$. The stiffness value is provided as the first UMAT parameter. 
+
+### Solution
+
+Under the load the beam bends and acts on the soil. The following picture shows the bended beam and soil displacement.
+
+![Displacement](interface_on_beam_deformation.svg)
+
+### Assertions
+
+The test asserts maximum values of displacement in X and Y directions. The values are compared with the correspondent solution obtained for a base case. The base case has the same settings, but it does not use the interface. 
+
+## Weak Interface
+
+### Setup
+
+This test uses the same setting as the previous test except the interface stiffness of $1e2 \  \mathrm{[Pa]}$. 
+
+### Solution
+
+This very small value of the stiffness leads to a large movement of the beam against the soil. The following picture shows that the beam is bended as in the previous test but the beam affects the soil displacement very little. The brown lines depict the interfaces' horizontal lines.
+
+![Displacement](week_interface_on_beam_deformation.svg)
+
+### Assertions
+
+The test checks that the beam displacement shall be much larger than the soil displacement. 
+
+## Interface cohesive side
+
+This test models a vertical movement of the soil block which left hand side is connected to the interface. 
+
+### Setup
+
+The model consists of 275 3-noded elements for the soil (using the UPwSmallStrainElement2D3N class) and 21 4-noded elements for interface (UPwSmallStrainInterfaceElement2D4N). A schematic representation can be found in the figure below, the interface is shown with a pink color. 
+
+![MeshStructure](box.svg)
+
+The soil is dry, and it is described with GeoLinearElasticPlaneStrain2DLaw:
+-   A Young's modulus $E = 1.0E9 \  \mathrm{[Pa]}$ with Poisson's ratio $\nu = 0.49 \  \mathrm{[-]}$.
+-   The soil density is $2000 \  \mathrm{[kg/m^3]}$ and the porosity is $n=0.3$. 
+-   The bulk modulus of solid $K = 1.0e12 \  \mathrm{[Pa]}$.
+
+The interface is described with SmallStrainUDSM2DInterfaceLaw and it has a cohesion of $1000 \  \mathrm{[kN]}$ and a stiffness of $1e12 \  \mathrm{[Pa]}$. 
+
+The left side of the interface is fixed. The soil moves down with a prescribed vertical displacement of $-0.1 \  \mathrm{[m]}$  and a horizontal line load of $-1 \  \mathrm{[kN]}$  is applied to the soil right side. 
+
+### Solution
+
+A result of the prescribed displacement is shown below. The soil moved down, when the interface is squeezed because its left side is fixed.  
+
+![Displacement](box-moved.svg)
+
+
+### Assertions
+
+The test checks the expected shear stress in the interface of $10 \  \mathrm{[kN]}$. 
```

Comment:
Do you know how we get to this number? Is there an analytical certain formula for the shear stress in this case, or is it more of a regression test?

Reply:
I do not know. It should be simple 'the output = the input'. Vahid wrote 'The interface has a cohesion of 10 kN.' However, I do not find where this is the input.

Reply:
Hmm okay, there is one parameter in the UMAT_PARAMETERS that's 10000 (could be 10 kN), it's the third of them in https://github.com/KratosMultiphysics/Kratos/blob/8814103c73864beff40e44b62bb87c04dc4e7760/applications/GeoMechanicsApplication/tests/Interface/interface_side_cohesive/MaterialParameters.json#L48. Could that be what we're looking for?

Reply:
You are right. I made more runs for other tests because they shall specific value for cohesion but materail files did not. Corrected README file.